test/schema-examples.test.js

```diff
@@ -431,7 +431,7 @@ test('Example - schemas examples', t => {
     }
   }
 
-  fastify.get('/', {
+  fastify.post('/', {
```

Comment:
Could you update the validation-and-serialization.md file?

This test file contains all the snippet that are in that document

Reply:
Yes, I will have a look at it tomorrow!

Reply:
I've looked through the docs but haven't found a snippet anywhere that I've changed that shows up in the docs. Can you show me what I should change?

Reply:
This test was checking the schemas only
https://github.com/fastify/fastify/blob/next/docs/Validation-and-Serialization.md#examples

not the schema+routing

so you are right: no update needed